ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Roadmap
+This document represents the current working roadmap for Pantheon.  It is a living document, which will evolve and change over time.  In particular the features in later versions are likely to be refined and change.
+
+We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  Our now scale is about 3 months, next about 6 months, and then later is beyond.
+
+## Now
+Our key three areas for now are:
+* Permissioning
+* First Class Client
+* iBFT 2.0
+
+### Permissioning
+We are implementing the key elements of an Enterprise Ethereum Permissioned network.  The initial version of this will be based around a JSON RPC API to manage the network. This will form the foundation for a smart contract based solution which will be developed in the `next` release (1.1)
+
+### First Class Client
+There is an ongoing piece of work underway enhancing the core performance of Pantheon, and ensuring that it behaves well as a first class client. The key elements of this are implementation of some performance benchmarks, finalising the options for the command line, and implementing an appropriate fast sync mechanism.
+
+### iBFT 2.0
+Work is underway designing and implementing a new PBFT consensus mechanism, improving the existing iBFT protocol, addressing some liveness and safety issues in the original protocol.
+
+## Next
+The key areas for next are:
+* Smart contract based permissioning
+* Privacy
+* Ethereum 1.x
+* iBFT 2.x
```

Comment:
Should there be a section below expanding on what this is (as for the other three bullet points)?